mmv1/third_party/terraform/tests/resource_firebase_extensions_instance_test.go.erb

```diff
@@ -16,12 +16,12 @@ func TestAccFirebaseExtensionsInstance_firebaseExtentionsInstanceResizeImageUpda
 	context := map[string]interface{}{
 		"project_id":    acctest.GetTestProjectFromEnv(),
 		"location":      "us-central1",
-		"random_suffix": RandString(t, 10),
+		"random_suffix": acctest.RandString(t, 10),
```

Comment:
Were these changes intended here (for `RandString` and `VcrTest`)?

Reply:
Thanks for catching it. These changes should not in this PR.